pennylane/devices/capabilities.py

```diff
@@ -79,6 +79,23 @@ def __and__(self, other: "OperatorProperties") -> "OperatorProperties":
         )
 
 
+def _supports_operator(op_name: str, op_dict: dict[str, OperatorProperties]) -> bool:
+    """Checks if the given operator is supported by name."""
+
+    if op_name in op_dict:
+        return True
+
+    if match := re.match(r"Adjoint\((.*)\)", op_name):
+        base_op_name = match.group(1)
+        return base_op_name in op_dict and op_dict[base_op_name].invertible
```

Comment:
How about `op**-1`? Should we also do a search for that?

Reply:
When we're talking about inverse, it that `op**-1` inverse or `adjoint(op)` "inverse".

Reply:
I think the `invertible` property refers to the `Adjoint` of an operator. At least that's how Catalyst treats it.